man/visualSubsetRecode.Rd

```diff
@@ -0,0 +1,60 @@
+\name{visualSubsetRecode}
+\alias{visualSubsetRecode}
+\title{Inspect data subsets visually and recode instantaneously}
+\description{
+This function allows manual inspection and manual recoding of subsets of the data.}
+\usage{visualSubsetRecode(dat, subsetInfo, ID, toRecodeVal="mci", useGroups=NULL)}
+\arguments{
+  \item{dat}{A data frame}
+  \item{subsetInfo}{A data frame. Colnames have to be: "datCols" and whatever is specified in ID and useGroups.}
```

Comment:
I would like to see a more detailed explanation of what these two data sets are and what they mean in terms of content.